groups/bsl/bslstl/bslstl_variant.h

```diff
@@ -287,11 +373,11 @@ BloombergLP::bslmf::MovableRef<
     typename variant_alternative<INDEX, variant<HEAD, TAIL...> >::type>
 get(BloombergLP::bslmf::MovableRef<variant<HEAD, TAIL...> >);
 #endif  // BSL_VARIANT_FULL_IMPLEMENTATION
-// Returns the contained alternative object in the supplied 'variant object
-// based on the index of the alternative type.  If the requested alternative
-// type is not the currently active alternative, an exception of type
-// 'bad_variant_access' is thrown.  The function will not compile if INDEX is
-// an invalid index.
+    // Returns the contained alternative object in the supplied 'variant object
+    // based on the index of the alternative type.  If the requested
+    // alternative type is not the currently active alternative, an exception
+    // of type 'bad_variant_access' is thrown.  The function will not compile
+    // if INDEX is an invalid index.
```

Comment:
The first sentence of a function contract must be in the imperative mood. Parameter names need to be introduced using the adjective "specified". Template parameter names should be indicated as such. Remember also to use the "shall" wording for mandates.
```suggestion
    // Return a reference to the alternative object at index (template
    // parameter) 'INDEX' in the specified 'obj'.  If 'INDEX' is not the index
    // of the currently active alternative, an exception of type
    // 'bad_variant_access' is thrown.  'INDEX' shall be a valid index for the
    // variant type of 'obj'.
```